### PR TITLE
Update WooCommerce Services extensions banner

### PR DESCRIPTION
**Pull request by v18**
Changes in this PR

- The banner image was updated
- Responsive styles were added
- Banner text was updated

I used your copy suggestion, @MarinaWoo, but I didn't mention the parts about tracking shipping for now. cc @MarinaWoo for another look :)

## Here's what it looks like now on on the featured page:

featured-page-big-screen

## and on the shipping page:

shipping-page-big-screen-2

## the shipping page on a slightly smaller screen:

shipping-page-big-screen-1

I know it doesn't quite line up with the other boxes on that page, but that would quite a bit of work. If there's a better look that doesn't rely on the width of the boxes, I'm all ears :)

## On a small screen for featured:

500px-featured

## On a small screen for shipping:

500px-shipping

## Here's a mid-way point for reference:
medium-screen-featured

@olaolusoga any final thoughts? :) cc @jeffstieler @nabsul @garymurray

### CODE REVIEW

File: includes/admin/class-wc-admin-addons.php [hunk 1]
```diff
@@ -332,30 +332,32 @@ public static function output_wcs_banner_block( $block = null ) {
 		);
 
 		$defaults = array(
-			'image'       => WC()->plugin_url() . '/assets/images/wcs-extensions-banner.png',
+			'image'       => WC()->plugin_url() . '/assets/images/wcs-extensions-banner-3x.png',
 			'image_alt'   => __( 'WooCommerce Services', 'woocommerce' ),
-			'title'       => __( 'Transform your store into a shipping and fulfillment machine', 'woocommerce' ),
-			'description' => __( 'WooCommerce Services makes shipping a breeze. Print a label, take advantage of discounted shipping rates, and send tracking information to your customer right as you process your order, all from the convenience of your WordPress dashboard.', 'woocommerce' ),
+			'title'       => __( 'Buy discounted shipping labels, print them from your dashboard.' ),
```
Comment (mikejolley) [inline includes/admin/class-wc-admin-addons.php]: Missing textdomain

Comment (mikejolley) [inline includes/admin/class-wc-admin-addons.php]: Should this be:

> Buy discounted shipping labels AND print them from your dashboard.

File: includes/admin/class-wc-admin-addons.php [hunk 2]
```diff
@@ -332,30 +332,32 @@ public static function output_wcs_banner_block( $block = null ) {
...
+			'description' => __( 'WooCommerce Services makes shipping a breeze. Get discounted rates, and print labels – all from the convenience of your WordPress dashboard.', 'woocommerce' ),
```
Comment (mikejolley) [inline includes/admin/class-wc-admin-addons.php]: No comma before 'and'

**Review by mikejolley — CHANGES_REQUESTED** [review]
Few changes needed. Can you sync with master too? May need to update npm dependencies so it doesn't minify everything.

File: includes/admin/class-wc-admin-addons.php (hunk 1 above)
Comment (v18) [inline includes/admin/class-wc-admin-addons.php]: Fixed

File: includes/admin/class-wc-admin-addons.php (hunk 2 above)
Comment (v18) [inline includes/admin/class-wc-admin-addons.php]: Fixed

**Comment by v18** [comment]
Fixed the issues above, and rebased.

> Can you sync with master too? May need to update npm dependencies so it doesn't minify everything.

Sure, I rebased on master and ran `grunt css` again.

File: includes/admin/class-wc-admin-addons.php (hunk 1 above)
Comment (v18) [inline includes/admin/class-wc-admin-addons.php]: > Should this be:

> Buy discounted shipping labels AND print them from your dashboard.

I think it _could_ be that. cc @olaolusoga and @MarinaWoo for word-smithing input.

**Pull request merged**